reports/REPORT.md

```
@@ -0,0 +1,19 @@
+# iOS CI Diagnosis
+
+## Most likely root cause
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)